# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome14/gnome-print10/freetype-update.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./libgnomeprint/gnome-font-private.h.orig	2004-12-20 21:26:06.517265736 +0100
+++ ./libgnomeprint/gnome-font-private.h	2004-12-20 21:26:40.597084816 +0100
@@ -22,7 +22,8 @@
 
 BEGIN_GNOME_DECLS
 
-#include <freetype/freetype.h>
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include <libgnomeprint/gp-character-block.h>
 #include <libgnomeprint/gp-fontmap.h>
 #include <libgnomeprint/gnome-font.h>
--- ./libgnomeprint/parseTT.h.orig	2004-12-20 21:27:18.782279784 +0100
+++ ./libgnomeprint/parseTT.h	2004-12-20 21:27:58.064308008 +0100
@@ -22,7 +22,8 @@
 BEGIN_GNOME_DECLS
 
 #include <glib.h>
-#include <freetype/freetype.h>
+#include <ft2build.h>
+#include FT_FREETYPE_H
 
 /*
  * Convert loaded Freetype TTF face to Type1
--- ./installer/gnome-font-install.c.orig	2004-12-20 21:28:44.275282872 +0100
+++ ./installer/gnome-font-install.c	2004-12-20 21:29:08.678573008 +0100
@@ -26,7 +26,8 @@
 #include <parser.h>
 #include <xmlmemory.h>
 /* End of ugly thing */
-#include <freetype/freetype.h>
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include <libgnome/gnome-defs.h>
 #include <libgnome/gnome-util.h>
 #include <libgnome/gnome-i18n.h>
--- ./installer/gf-pfb.c.orig	2004-12-20 21:29:39.042956920 +0100
+++ ./installer/gf-pfb.c	2004-12-20 21:30:02.348413952 +0100
@@ -12,7 +12,8 @@
 #include <fcntl.h>
 #include <string.h>
 #include <ctype.h>
-#include <freetype/freetype.h>
+#include <ft2build.h>
+#include FT_FREETYPE_H
 #include "gf-pfb.h"
 
 #define noVERBOSE
--- ./installer/gf-ttf.c.orig	2004-12-20 21:30:43.089220408 +0100
+++ ./installer/gf-ttf.c	2004-12-20 21:31:16.921077184 +0100
@@ -32,7 +32,8 @@
 #include <fcntl.h>
 #include <string.h>
 #include <ctype.h>
-#include <freetype/freetype.h>
+#include <ft2build.h>
+#include FT_FREETYPE_H
 
 #include "gf-ttf.h"
 
